### DISCUSSION TITLE
New article about the Tinfour contouring API

### DISCUSSION THREAD

**Discussion by gwlucastrig** [gwlucastrig/Tinfour#73] [General]
I've been working on a new wiki article that describes how to use the Tinfour contouring API.  If you would like, you can view it at https://github.com/gwlucastrig/Tinfour/wiki/Using-The-Tinfour-Contouring-API

I'm still polishing the article, so it may change a little bit over the next several days.  If you see anything that needs fixing or improvement, please let me know.

**Comment by micycle1** [comment]
Much superior to the previous article. I hadn't known of `ContourUtilities.computeContourValues()` or `SmoothingFilter` until now.
